### PR TITLE
[11.0][MIG] account_invoice_tax_required: Migration to 11.0

### PR DESCRIPTION
**Pull request by jctrey**
Moved from OCA/account-financial-tools

### CODE REVIEW

File: account_invoice_tax_required/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
```
Comment (oihane) [inline account_invoice_tax_required/README.rst]: It shoul be 
```
   :target: https://www.gnu.org/licenses/agpl
```

File: account_invoice_tax_required/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+#. Go to Accounting (or Invoicing) > Sales > Customer Invoices.
+#. Create a new invoice.
+#. Insert a new invoice line without taxes.
+#. Press on "Validate".
+#. An error will raise preventing to do it.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/92/10.0
```
Comment (oihane) [inline account_invoice_tax_required/README.rst]: Wrong version number

File: account_invoice_tax_required/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2015 - Camptocamp SA - Author Vincent Renaville
+# Copyright 2016 - Tecnativa - Angel Moya <odoo@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': "Tax required in invoice",
+    'version': "11.0.1.0.0",
+    "author": "Camptocamp,Tecnativa,Odoo Community Association (OCA)",
+    'website': "http://www.camptocamp.com",
+    'category': "Localisation / Accounting",
```
Comment (oihane) [inline account_invoice_tax_required/__manifest__.py]: misspelled "localization"

**Review by oihane — APPROVED** [review]
Minor changes, and I think that you should remove *.pot file.
But I think that it's correct everything else.

File: account_invoice_tax_required/README.rst (hunk 1 above)
Comment (jctrey) [reply account_invoice_tax_required/README.rst]: Changed

File: account_invoice_tax_required/README.rst (hunk 2 above)
Comment (jctrey) [reply account_invoice_tax_required/README.rst]: Changed

File: account_invoice_tax_required/__manifest__.py (hunk 3 above)
Comment (jctrey) [reply account_invoice_tax_required/__manifest__.py]: Changed

File: account_invoice_tax_required/__init__.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Copyright 2015 - Camptocamp SA - Author Vincent Renaville
+# Copyright 2016 - Tecnativa - Angel Moya <odoo@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (etobella) [inline account_invoice_tax_required/__init__.py:3]: https

**Review by etobella — APPROVED** [review]
LGTM :+1: 
Just a comment

**Review by rgarnau — APPROVED** [review]
LGTM :+1:

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**